deployment/ccip/config/rmn.go

```diff
@@ -0,0 +1,39 @@
+package config
```

Comment:
does it make sense to move it under https://github.com/smartcontractkit/chainlink/blob/9d8d9f48b92cab72de51e4484cc7b0833b1ccf6b/deployment/ccip/changeset/cs_update_rmn_config.go ? It will help locate all pieces together

Reply:
Yeah good idea, I will make the change

Reply:
Also it would be nice if you can get the test to use these configs